### DISCUSSION TITLE
Trying to create an external page

### DISCUSSION THREAD

**Discussion by rosalinet** [nunocoracao/blowfish#1012] [Q&A] [answered]
Hey all, I'm using [these instructions](https://blowfish.page/docs/content-examples/) (External links section) to create a menu item that will redirect to an external page, but it keeps showing up as an empty page. 
In my menus.toml file I have:
<img width="462" alt="image" src="https://github.com/nunocoracao/blowfish/assets/92281462/07546d93-e7d0-47d6-a604-1895e5b5100c">
and then in my content folder, I have a folder called 'documentation', and within that folder, my index.md file looks like this:
<img width="549" alt="image" src="https://github.com/nunocoracao/blowfish/assets/92281462/4a9f57b0-4d29-4ed9-bed4-d6c13790e3d9">
The structure should be completely fine since I have another page that doesn't redirect to an external link in that content folder, and that internal page works completely fine.

Any help? Also, let me know if you need more info. Thanks!

**Comment by JahsehJ** [comment, marked as answer]
in the menu config, just set url and get rid of pageref, that'd take you to the url you set

**Reply by JahsehJ** [reply]
the external page feature is only useful if you want to display an external link on list pages ^
with menus just set the url in the config
